mayastor-test/e2e/install/install_test.go

```diff
@@ -113,8 +113,8 @@ func getDeployYamlDir() string {
 func applyDeployYaml(filename string) {
```

Comment:
At some point we should move these functions to the e2e/common/ directory.

Reply:
agree. It should be done at some point (in a future PR).